data/transport missions.txt

```diff
@@ -1011,6 +1040,24 @@ mission "Rim Archaeology 1"
 			scene "scene/geoscan"
 			`	"Great burning suns!" says Foster. "There is something down there! Roads in a triangular grid. That ain't no human city." You both stare at the monitor for a minute, then Foster says, "I'm going to need to pull some permits. Meet me in the spaceport in an hour if you're willing to help pick up some of the other stuff we'll need."`
 
+mission "logbook: sad archie before sayaiban"
+	invisible
+	to offer
+		not "spoke with archon"
+		has "Rim Archaeology 1: done"
+	on offer
+		log `Talked to Albert Foster about my dream with the dragon-people, and now he has me convinced I was spoken to by a telepathic space alien called an Archon. Albert has found some sort of ancient city on Zug and will be excavating it soon. Hopefully this isn't one big drug trip.`
```

Comment:
and now he has me convinced I was spoken to by a telepathic space alien called an Archon -> and he says that it was a telepathic space alien called an Archon that spoke to me.